Cue/app/tabs/library/LibraryHome.js

```diff
@@ -125,6 +126,44 @@ class LibraryHome extends React.Component {
   }
 
   _onPressAddDeck = () => {
+    Alert.alert(
+      Platform.OS === 'android' ? 'Add deck' : 'Add Deck',
+      'You can create a new private deck or add a shared deck by entering a share code.',
+      [
+        {text: 'Cancel', style: 'cancel'},
+        {text: 'Look up share code', onPress: this._onPressLookUpDeckByShareCode},
+        {text: 'Create new deck', onPress: this._onPressCreateDeck},
```

Comment:
Is it ok to just have the string literal in title case since android will capitalize it anyway?